inc/Engine/Optimization/GoogleFonts/CombineV2.php

```diff
@@ -105,15 +100,12 @@ public function optimize( string $html ): string {
 	 * Parses found matches to extract fonts and subsets.
 	 *
 	 * @since  3.8
-	 *
-	 * @param array $tag A Google Font v2 url.
-	 *
-	 * @return bool
+	 * @param  array $tag A Google Font v2 url.
+	 * @return false|array
```

Comment:
instead of returning `false`, we can return an empty array, that way the method only as one return type